packages/tasks/src/tasks/text-generation/data.ts

```diff
@@ -14,6 +14,10 @@ const taskData: TaskDataCustom = {
 			description: "Truly open-source, curated and cleaned dialogue dataset.",
 			id: "HuggingFaceH4/ultrachat_200k",
 		},
+		{
+			description: "A reasoning dataset.",
+			id: "open-thoughts/OpenThoughts-114k",
```

Comment:
```suggestion
			id: "open-r1/OpenThoughts-114k-math",
```

Filtered + enriched version of the former - plus gives some vis to the `open-r1` org